website/content/docs/commands/credentials/read.mdx

```diff
@@ -1,8 +1,8 @@
 ---
 layout: docs
 page_title: credentials read - Command
-description: |-
-  The "credentials read" command reads information for a credential given its ID.
+description: >-
+  The "credentials read" command retrieves the details of an existing credential resource given its ID.
```

Comment:
there isn't a consistent use of the "given ID" terminology throughout, but I'd at least change this one

```suggestion
  The "credentials read" command retrieves the details of an existing credential resource for a given ID.
```